### DISCUSSION TITLE
How do I prevent my own styles bleeding into autodocs?

### DISCUSSION THREAD

**Discussion by tillsanders** [storybookjs/storybook#23254] [Help]
I'm using Storybook 7.0.24 for vue3-vite. I have some custom styles that I'm importing in `preview-head.html`. That works fine in general, but my own styling "bleeds" into the Autodoc and messes with Storybooks own components / DocBlocks.

Is there any way to prevent this?

**Comment by codyburleson** [comment]
I have a similar, or the same, issue. As shown in the attached image, I have styles bleeding into the code block, which is messing up the whole look of the code example:

![image](https://github.com/storybookjs/storybook/assets/7317733/d6310c88-e202-4fe1-b9aa-bd7bbca017a1)

I'm using Storybook 7.5.3 for web-components-vite

It seems that, for some reason, one of the stylesheets we're using inside of components, is being injected into the HTML head...

![image](https://github.com/storybookjs/storybook/assets/7317733/6f5b8b23-4744-4ba7-9ca7-264dbae84cfa)
